src/netstandard/netstandard.targets

```diff
@@ -0,0 +1,27 @@
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="_BuildNetstandardFacade">
+   <MSBuild Projects="..\Mono.Android\Mono.Android.csproj"
```

Comment:
Please see the `System.Drawing.Primitives.targets` comments and apply here as well.